schemas/list-workertypes-response.yml

```diff
@@ -20,10 +20,45 @@ properties:
           minLength:    {$const: identifier-min-length}
           maxLength:    {$const: identifier-max-length}
           pattern:      {$const: identifier-pattern}
+        provisionerId:
+          title:        "Provisioner ID"
+          type:         string
+          minLength:    {$const: identifier-min-length}
+          maxLength:    {$const: identifier-max-length}
+          pattern:      {$const: identifier-pattern}
+        stability:
+          title:        "Stability"
+          description: |
+            The stability of the worker-type. Accepted values:
+            'experimental|stable|deprecated'
+          type:         string
+          enum:         ["experimental", "stable", "deprecated"]
+        description:
+          title:        "Description"
+          description: |
+            Description of the worker-type.
+          type:         string
+        documentation:
```

Comment:
This is the list resource.  Would it make sense to omit documentation and payload schema here, but include them a get-workertype endpoint?  So, same REST resource, just different views?